src/servlets/utils/constants.ts

```diff
@@ -28,6 +28,25 @@ export const SIGNUP_REF_IMAGE_URL =
   'https://download.audius.co/static-resources/signup_referral.png'
 export const USER_NODE_IPFS_GATEWAY = 'https://usermetadata.audius.co/ipfs/'
 
+// App routes that should not be treated as user handles
+export const APP_ROUTES = [
```

Comment:
maybe should be everything from here?

https://github.com/AudiusProject/audius-protocol/blob/29dc04e5c0089ec6b3469b427fb9f1bad0eb168b/packages/common/src/utils/route.ts#L1

Reply:
yeah i referenced that exact file

Reply:
ill do another check though

Reply:
nice yeah added more, also dropping the exploreMap heads up